homeassistant/components/rfxtrx/binary_sensor.py

```diff
@@ -96,7 +98,7 @@ def get_sensor_description(type_string: str):
 
     def _constructor(
         event: rfxtrxmod.RFXtrxEvent,
-        auto: bool,
+        auto: rfxtrxmod.RFXtrxEvent | None,
```

Comment:
This did not correspond to the hint defined in `__init__.py`
I think the `__init__.py` method should be adjusted to pass a `bool`, but that would be a code change and it seemed safer to adjust the type hint instead to match the current arguments.

https://github.com/home-assistant/core/blob/67c35652f0d49dc56e1e934fab3b2c51f5f82592/homeassistant/components/rfxtrx/__init__.py#L249-L252
and
https://github.com/home-assistant/core/blob/67c35652f0d49dc56e1e934fab3b2c51f5f82592/homeassistant/components/rfxtrx/__init__.py#L290

Reply:
The second event is a auto detected event. So this change is correct. A refactor for auto might be in order later.